examples/datatype-macro.kl

```diff
@@ -3,6 +3,7 @@
 (import (shift "n-ary-app.kl" 1))
 (import (shift "lispy-do.kl" 1))
 (import (shift "let.kl" 1))
+(import (shift "quasiquote.kl" 1))
```

Comment:
huh, why was that import not previously needed even though the file uses `quasiquote/loc`? Presumably, that same mysterious reason is also why `free-identifier-case-test.kl` and `examples/mcond-test.kl` now import `n-ary-app.kl`?

Reply:
This is due to #48 getting fixed. Modules were transitively exporting their dependencies' exports! Fixing that meant writing exports correctly.